README.md

```diff
@@ -1,17 +1,119 @@
 # chisel-testers2
-See [freechipsproject/chisel3#725](https://github.com/freechipsproject/chisel3/issues/725).
-Discussion should happen on the issue above.
+This is **experimental software** that is currently under development, and interfaces may be subject to change (see [stability](#stability) for details).
```

Comment:
I think it might be better to just say alpha here (or maybe beta) those are more traditional terms for new software

Reply:
changed